pkg/scheduler/preemption/preemption.go

```diff
@@ -395,6 +403,28 @@ func workloadFits(wlReq cache.FlavorResourceQuantities, cq *cache.ClusterQueue,
 	return true
 }
 
+// is queue exhausted for all requested flavors
+func isQueueExhaustedForAllRequestedFlavors(wl *workload.Info, assignment flavorassigner.Assignment, cq *cache.ClusterQueue) bool {
```

Comment:
```suggestion
func isQueueExhaustedInAnyRequestedFlavors(wl *workload.Info, assignment flavorassigner.Assignment, cq *cache.ClusterQueue) bool {
```

if I'm interpreting the logic correctly.

Reply:
That's right. I have updated it, and added some test that demonstrate a difference between "any" and "all", like "minimal preemptions when target queue is exhausted in one resource, but not the other".